controllers/networkinterface_controller.go

```diff
@@ -47,36 +45,19 @@ const (
 	defaultFirewallRulePrefix = "0.0.0.0/0"
 )
 
-// workaroundNoNetworkInterfaceIPV6 is a workaround to only use ipv4 addresses.
-// remove as soon as bug is fixed.
-func workaroundNoNetworkInterfaceIPV6(ips []netip.Addr) []netip.Addr {
-	var res []netip.Addr
-	for _, ip := range ips {
-		if ip.Is4() {
-			res = append(res, ip)
-		}
-	}
-	return res
-}
-
-func workaroundRequiredIPv6Address(ip netip.Addr) netip.Addr {
-	if ip.IsValid() {
-		return ip
-	}
-
-	var data [16]byte
-	_, err := crand.Read(data[:])
-	utilruntime.Must(err)
-
-	return netip.AddrFrom16(data)
-}
-
 func getIP(ipFamily corev1.IPFamily, ipFamilies []corev1.IPFamily, ips []metalnetv1alpha1.IP) netip.Addr {
 	for i, family := range ipFamilies {
 		if ipFamily == family {
 			return ips[i].Addr
 		}
 	}
+	// If an IP is not specified defined it as "::" or "0.0.0.0" to disable that ip family on dp-service side
```

Comment:
It is a bit difficult to digest this comment. Is it a TODO or something?

Reply:
Yeah, there is also a typo in the comment. I will reword it.